hikari/api/special_endpoints.py

```diff
@@ -982,11 +982,23 @@ def default_permission(self) -> undefined.UndefinedOr[bool]:
         """Whether the command should be enabled by default (without any permissions).
 
         Defaults to `builtins.bool`.
+        """
 
-        Returns
-        -------
-        undefined.UndefinedOr[builtins.bool]
-            Whether the command should be enabled by default (without any permissions).
+    @property
+    @abc.abstractmethod
+    def default_member_permissions(self) -> undefined.UndefinedOr[permissions_.Permissions]:
+        """Member permissions necessary to utilize this command by default.
+
+        If `0`, then it will be disabled by default. This excludes administrators
```

Comment:
But really an UNDEFINED in this case should be the default `@everyone` permission without overwrites. Unless I'm misunderstanding you?

Reply:
I see. `0` is admins, etc. with the API correct? So you're suggesting to hide this behavior and take `8` for admins instead?

Reply:
> Did you ever consider trying to hide this behaviour on a library level considering that it just doesn't really make any sense since it just makes setting it to 0 equal to setting it to 8 while leaving setting it to UNDEFINED as the real 0?

Yea, that is exactly what it is at the moment. Its just the edge case of 0 basically meaning 8 that I would document (as have done Discord)

Reply:
> > Did you ever consider trying to hide this behaviour on a library level considering that it just doesn't really make any sense since it just makes setting it to 0 equal to setting it to 8 while leaving setting it to UNDEFINED as the real 0?
> 
> Yea, that is exactly what it is at the moment. Its just the edge case of 0 basically meaning 8 that I would document (as have done Discord)

> Did you ever consider trying to hide this behaviour on a library level

Reply:
Honestly, I don't think it's a good idea as it abstracts too much from the API docs and info. Even though technically it should be the same, it could be confusing when comparing the behavior of Hikari vs discord API docs.

Reply:
After revisiting this PR this much time later I have realized how funky that behavior could be and we might as well account for it ourselves. Changed the behavior to account for Discord's interpretation of `default_member_permissions` being 0.